django/db/models/fields/tuple_lookups.py

```diff
@@ -313,19 +324,26 @@ def process_rhs(self, compiler, connection):
                 )
             )
 
+        # SQLite requires an explicit `VALUES` declaration before
+        # the tuples.
+        if connection.vendor == "sqlite":
+            result[0].function = "VALUES"
```

Comment:
I wonder if there is a way to avoid vendor checks in `get_fallback_sql()` :thinking:

Reply:
I guess we could add a feature flag like SQL alchemy did https://github.com/sqlalchemy/sqlalchemy/issues/4766

Reply:
What about doing this in `Tuple.as_sqlite()`?

Reply:
I haven't tried it out but it's worth giving it a shot for sure.

Reply:
I think I pushed a more elegant solution by introducing `TupleList(Tuple).as_sqlite` @felixxm. Let me know what you think.